hash.c

```diff
@@ -5844,8 +5844,8 @@ env_f_to_hash(VALUE _)
 
 /*
  * call-seq:
- *   ENV.to_h                        -> hash
- *   ENV.to_h {|name, value| block } -> hash
+ *   ENV.to_h                        -> Hash
+ *   ENV.to_h {|name, value| block } -> Hash
```

Comment:
Why `Hash`?
It looks like that the `Hash` class itself will be returned.

Reply:
I count 13 "-> Hash" vs. 3 "-> hash", so thought to make consistent.  But I probably should not have mixed that in with the definite doc errors.

Reply:
Do you want all Hash, all hash, or leave unchanged?